### PR TITLE
Angular update

### PR DESCRIPTION
**Pull request by sei-ebram**
- Update from Angular 9 to Angular 11
- Update all package versions to latest

### CODE REVIEW

File: tslint.json [hunk 1]
```diff
@@ -73,7 +73,6 @@
     "no-trailing-whitespace": true,
     "no-unnecessary-initializer": true,
     "no-unused-expression": true,
-    "no-use-before-declare": true,
```
Comment (sei-aschlackman) [inline tslint.json:76]: Why remove this?

Comment (sei-ebram) [reply tslint.json:76]: that's all done as part of the Angular 10 upgrade:
https://github.com/angular/angular-cli/issues/17117

Comment (sei-aschlackman) [reply tslint.json:76]: Ok, cool.

**Review by sei-aschlackman — APPROVED** [review]

**Review by sei-aschlackman — APPROVED** [review]

**Comment by sei-ebram** [comment]
tested successfully in development environment

**Review by sei-noconnor — APPROVED** [review]

**Pull request merged**